app/views/development_metrics/_modal_pull_requests.erb

```diff
@@ -0,0 +1,14 @@
+<div id="modal-popup" class="modal hide fade" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true"
+  data-urls="<%= request_urls.to_json %>">
+  <div class="modal-dialog modal-dialog-centered modal-xl" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title" id="modal-title">Selection Details</h5>
```

Comment:
To know, do we have standardized sizes and types of texts? Because I am wondering if it's not better to put this into a div and to add a class with the styles but I am not sure 😅